reliquery/tests/test_storage.py

```diff
@@ -224,3 +226,62 @@ def test_different_storage_types_given_env_config(mockholder, tmpdir):
             assert stor.name == "s3"
         elif isinstance(stor, DropboxStorage):
             assert stor.name == "dropbox"
+
+
+# GoogleDrive
+def GetMockedGoogle(root):
+    list = Mock()
+
+    list.execute.return_value = {"files": [{"name": root, "id": 1}]}
+
+    files = Mock()
+    files.list.return_value = list
+
+    service = Mock()
+    service.files.return_value = files
+
+    return GoogleDriveStorage(
+        "relics",
+        "test1",
+        "/Users/jacqueline/Downloads/token.json",
+        ["https://www.googleapis.com/auth/drive"],
+        "1apx2fBZ-IF7QeBZentqfLY8hZe1OsHyN",
+        service=service,
+    )
+
+
+@mock.patch("reliquery.storage.GoogleDriveStorage._list_items_in_folder")
+def test_google(_list_items_in_folder):
```

Comment:
you may want to change the names to something slightly more descriptive than test_google{2,3}